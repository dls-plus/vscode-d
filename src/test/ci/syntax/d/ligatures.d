auto a = () => x;
struct X(T...) {}