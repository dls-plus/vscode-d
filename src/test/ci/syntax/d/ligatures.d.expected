[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"variable.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":8,"scope":"keyword.operator.assign.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":12,"scope":"source.d"},{"start":12,"end":14,"scope":"keyword.operator.lambda.d"},{"start":14,"end":15,"scope":"source.d"},{"start":15,"end":16,"scope":"variable.d"},{"start":16,"end":18,"scope":"meta.statement.end.d"}]
[{"start":0,"end":6,"scope":"storage.type.struct.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":8,"scope":"entity.name.type.struct.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"variable.parameter.d"},{"start":10,"end":13,"scope":"support.type.tuple.d"},{"start":13,"end":14,"scope":"source.d"},{"start":14,"end":18,"scope":"source.d"}]